apps/docs/app/ui/components/componentExample/ComponentPreviewWrapper.tsx

```diff
@@ -44,11 +44,11 @@ const ComponentPreviewWrapper = memo(({ preview, toggleButton, minHeight = "13re
                     colorMode={localColorMode}
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/22cb1b1b-22ef-48ff-b399-644acd246293)
I would center these.

Reply:
the point is to not be centered. Later one, we specifically show what centering looks like. This is the default alignment of Flex.